prov/efa/src/dgram/efa_dgram_msg.c

```diff
@@ -326,7 +326,7 @@ ssize_t efa_dgram_post_flush(struct efa_dgram_ep *ep, struct ibv_send_wr **bad_w
 #if HAVE_LTTNG
 	struct ibv_send_wr *head = ep->base_ep.xmit_more_wr_head.next;
 	while (head) {
-		efa_tracing(post_send, (void *) head->wr_id);
+		efa_tracepoint_wr_id_post_flush((void *) head->wr_id);
```

Comment:
`post_flush` -> `post_send`

Reply:
I changed it to `post_flush` since those were the only functions this tracepoint was used, but I'm indifferent so I'll change it back.